controllers/controller.go

```diff
@@ -401,6 +401,14 @@ func GetConfigMap(context ctx.Context, cluster *fdbtypes.FoundationDBCluster, ku
 	}
 	data["sidecar-conf"] = string(sidecarConfData)
 
+	if cluster.Spec.ConfigMap != nil && len(cluster.Spec.ConfigMap.Data) > 0 {
+		items, err := json.Marshal(cluster.Spec.ConfigMap.Data)
+		if err != nil {
+			return nil, err
+		}
+		data["items"] = string(items)
```

Comment:
Instead of putting these in a special "items" key, can we add them directly into the ConfigMap? For instance, in the test case below, I think that the "itemVal" value should be in `configMap.Data["itemKey"]`.

Reply:
Done. thanks